sys-apps/smartmontools/smartmontools-6.5.ebuild

```diff
@@ -91,7 +91,7 @@ pkg_postinst() {
 			ewarn "database by: "
 
 			if ! use update_drivedb ; then
-				ewarn "re-merging ${PN} with USE='update_drivedb, then"
```

Comment:
you added these errors in a previous commit.  please revert/squash so these are never a problem.